zadaci/sa-kolokvijuma/2020/RA/T5/G8/zadatak.rst

```diff
@@ -0,0 +1,57 @@
+Iz zadate ulazne datoteke učitati podatke u jednostruko spregnutu listu. Podatke učitati u 
+rastućem redosledu po rednom broju rezervisanog sedišta.
+Struktura ``balet`` sadrži sledeća polja:
+
+- redni broj rezervisanog sedišta (pozitivna celobrojna vrednost)
+- naziv baleta (jedna reč, do 8 karaktera)
+- rezervisao (jedna reč, do 25 karaktera)
+- cena (pozitivna celobrojna vrednost)
+
+Takođe, struktura ``balet`` sadrži i polja potrebna za pravilno formiranje jednostruko spregnute 
+liste.
+
+Od korisnika zatražiti da unese naziv baleta (jedna reč, do 8 karaktera) i za uneti balet na 
+terminal ispisati naziv baleta, ukupan broj prodatih karata i ukupnu cenu karata. Ispisati i 
+podatke o rezervisanom sedištu, kupcu koji je rezervisao i ceni karte (ispoštovati očekivani izlaz prikazan u stavci pod rednim brojem 4.). 
+
+Napomene:
+
+- Prilikom unosa naziva baleta od strane korisnika voditi računa o tome da program treba da prepozna naziv baleta bez obzira na kombinaciju malih i velikih slova koje je korisnik uneo (pogledati stavku pod rednim brojem 2. npr. ukoliko korisnik unese „zizela”, u izlaznu datoteku treba ispisati informacije o prepoznatom baletu sa nazivom „Zizela”).
+- Kroz argumente komandne linije proslediti naziv ulazne datoteke.
+- Ukoliko za unete podatke od strane korisnika ne postoje podaci očekivan izlaz je prikazan u stavci pod rednim brojem 5.
+
+1. Primer poziva::
+
+   ./a.out balet.txt
+
+2. Unos naziva baleta preko terminala::
+
+    Naziv baleta: zizela
+
+3. Zadati ulaz u datoteci balet.txt::
+
+    5 Zizela Nadezda 310
+    11 Kopelija Nikola 230
+    23 Kopelija Milica 150
+    3 Zizela Igor 310
+    9 Pinokio Aleksandar 330
+    10 Zizela Ilija 250
+    22 Kopelija Milan 150
+    11 Zizela Stefan 250
```

Comment:
Code emphasis the filename (put it like this \`\`balet.txt\`\`), remove one colon (:) at the end of line 31 and instead of lines 33-40 use:

``.. literalinclude:: balet.txt``